Library/Homebrew/cask/installer.rb

```diff
@@ -156,7 +156,10 @@ def install
 
       oh1 "Installing Cask #{Formatter.identifier(@cask)}"
       # GitHub Actions globally disables Gatekeeper.
-      opoo_outside_github_actions "macOS's Gatekeeper has been disabled for this Cask" unless quarantine?
+      unless quarantine?
+        opoo_outside_github_actions "--no-quarantine bypasses macOS’s Gatekeeper, reducing system security. " \
+                                    "Do not use this flag unless you understand the risks."
```

Comment:
This wording feels a little late, saying "Do not use this flag unless you understand the risks" only after the flag has already been invoked might not be very helpful.